### PR TITLE
Moved from AzureRm to Az

### PR DESCRIPTION
**Pull request by jianyunt**

### CODE REVIEW

**Comment by edyoung** [comment]
AzurePsDrive won't be usable by anyone else until Az module is released. I don't think that's a blocking issue but let me know what you think

File: AzurePSDrive.psm1 [hunk 1]
```diff
@@ -4,25 +4,25 @@ using module .\AzurePSDriveStorageAccount.psm1
 using module .\AzurePSDriveVM.psm1
 using module .\AzurePSDriveWebApp.psm1
 
-$script:AzureRM_Profile = if($IsCoreCLR){'AzureRM.Profile.NetCore'}else{'AzureRM.Profile'}
-$script:AzureRM_Resources = if($IsCoreCLR){'AzureRM.Resources.Netcore'}else{'AzureRM.Resources'}
-$script:pathPattern = [System.IO.Path]::Combine('Azure:', '*', 'ResourceGroups', '*')
+$script:Az_Profile = 'Az.Profile'
+$script:Az_Resources = 'Az.Resources'
+$script:pathPattern = [System.IO.Path]::Combine('Az:', '*', 'ResourceGroups', '*')
```
Comment (edyoung) [inline AzurePSDrive.psm1]: Does this change the name of the root directory shown to the user?

Comment (jianyunt) [reply AzurePSDrive.psm1]: Actually this is the question I would like your guys today.  Should we take this chance to change the `Azure` drive to `Az` as well?  I made the change already. But if we don't think this is a good idea, I will back it out easily.

**Comment by jianyunt** [comment]
@edyoung You are right. AzurePSDrive won't be usable until the az modules ready for full and core clrs. We have to wait for that before we can publish it.

File: Tests/AzurePSDrive.Tests.ps1 [hunk 2]
```diff
@@ -36,15 +36,15 @@ $osDiskName = $VMName + "OSDisk"
...
+    -or (-not (Get-Module -Name $script:Az_Compute)) `
+    -or (-not (Get-Module -Name $script:Az_Network)) `
+    -or (-not (Get-Module -Name $script:Az_Storage)) `
     -or (-not (Get-Module -Name SHiPS)) `
     -or (-not (Get-Module -Name AzurePSDrive)))
     {
-        throw "Ensure $script:AzureRM_Profile, $script:AzureRM_Resources, $script:AzureRM_Compute, $script:AzureRM_Network, $script:AzureRM_Storage, SHiPS, AzurePSDrive modules are installed"
+        throw "Ensure $script:Az_Profile, $script:Az_Resources, $script:Az_Compute, $script:Az_Network, $script:Az_Storage, SHiPS, AzurePSDrive modules are installed"
     }
 }
```
[the remark this answers is not in the document]
Comment (jianyunt) [reply Tests/AzurePSDrive.Tests.ps1]: fixed. Good catch! Thank you.

**Comment by jianyunt** [comment]
It looks like the Az modules for `core` are published in the PS Gallery already.  The tests will fail because they are running on Windows Desktop.

**Comment by jianyunt** [comment]
Once Az modules for desktop published. Tests will be fine.

**Pull request merged**